File: src/main/java/com/jakeesveld/zoos/service/ZooService.java
package com.jakeesveld.zoos.service;

import com.jakeesveld.zoos.model.Zoo;
import com.jakeesveld.zoos.view.ZooDetails;

import java.util.ArrayList;

public interface ZooService {

    ArrayList<ZooDetails> getZooDetails();
    void updateZoo(long id, Zoo updatedZoo);
    void addZoo(Zoo zoo);
    void deleteZoo(long id);
    ArrayList<Zoo> findAll();
}

File: src/main/java/com/jakeesveld/zoos/service/ZooService.kt
package com.jakeesveld.zoos.service

import com.jakeesveld.zoos.model.Zoo
import com.jakeesveld.zoos.view.ZooDetails

import java.util.ArrayList

interface ZooService {

    val zooDetails: MutableList<ZooDetails>
    fun updateZoo(id: Long, updatedZoo: Zoo)
    fun addZoo(zoo: Zoo)
    fun deleteZoo(id: Long)
    fun findAll(): List<Zoo>
}
